templates/repository/common/CONTRIBUTING.md

```diff
@@ -139,34 +166,32 @@ Normally, all pull requests must include tests that test your change.
 Occasionally, a change will be very difficult to test for. In those cases,
 please include a note in your commit message explaining why.
 
-## How We Organize Our Work
+### Working with Forks
 
-All repositories in the [ORY organization](https://github.com/ory) have their issues and pull requests
-monitored in the [ORY Monitoring Board](https://github.com/orgs/ory/projects/9). This allows
```

Comment:
I found `How We Organize Our Work` section useful. Why it was removed? Is it because it is not used anymore? [ORY Monitoring Board](https://github.com/orgs/ory/projects/9) is empty?

Reply:
Yea the original monitoring board is not used anymore, we have however this [Maintainers Board](https://github.com/orgs/ory/projects/8) 
Apart from that current issues in Kratos for example get tracked with the [corp/m3+ ect. tags](https://github.com/ory/kratos/labels/corp%2Fm3).
As to why it is not used anymore im not 100% sure but I think it was too much work to maintain and the devs decided to track progress with the issue tags.